docs/usage.rst

```diff
@@ -1,5 +1,5 @@
-Using the API
-=============
+Using the Standard Client
```

Comment:
nit: standard client -> sync client?

Reply:
Eventually there will be two sync clients, so we'll need a different naming system. I'll change it to sync for now though, we can discuss that later!